src/pages/MarketPage/AddonDetails/AddonDetails.styled.js

```diff
@@ -175,6 +175,10 @@ export const MetaPanelRow = styled.div`
   flex-direction: column;
   gap: var(--base-gap-small);
 
+  .link {
+    text-decoration: underline;
```

Comment:
I have been experimenting with hover effects, I feel it needs something extra.

What do you think about these two? My favourite is the top one, acts more like a button.

https://github.com/user-attachments/assets/e3f4e06b-acc0-4fb1-ae8b-61b7693438b0

The icon is `open_in_new` and hover color is `var(--md-sys-color-secondary-container-hover)`.

You might want to use a Button with`variant="tonal"`. Then some custom CSS.. 

- Underline
- No BG when not hovering
- Width 100% and space-between text and icon
- Text color always on-surface
- `padding: 4px 6px;`

Reply:
👍 I'll use the top one then, I like it too.